src/Machine.js

```diff
@@ -256,7 +259,12 @@ class Machine {
 
     // read
     this.logger.info(`Including source "${includePath}"`);
-    const content = reader.read(includePath);
+    let content = reader.read(includePath);
+
+    // if content don't have line separator at the end, then add it
+    if (content[content.length - 1] != '\n') {
```

Comment:
Not necessary https://stackoverflow.com/questions/20916442/negative-indices-in-array